.github/workflows/cmake_install.yaml

```
@@ -25,16 +25,16 @@ jobs:
 
     strategy:
       matrix:
-        python_version: ['3.8.13', '3.9.13', '3.10.5']
+        python_version: ['3.9.13', '3.10.5', '3.13.0']
```

Patch versions are not needed here. I'd suggest removing them so we are sure to get the latest version of each Python 3.x version we want to support. I created #168 to track all of the workflows that need this, so we can delegate the full change to another PR if you don't want to do it here.
```suggestion
        python_version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```